resources/todomvc/architecture-examples/react/standalone/webpack.common.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    entry: {
+        app: "./standalone/src/index.js",
```

Comment:
It feels weird that these files are inside the standalone folder but the root folder for the execution is outside. But lets wait until the official review, since seems like  a matter of styling/preference